Header header
xhab/Temperature temp_data
xhab/FluidPressure pressure_data
xhab/FluidLevel fluid_lvl_data
xhab/ValveState valve_data
xhab/PumpState pump_data
int32 moisture_data
int32 ph_data
int32 ec_data
